dist/scss/wds-mixins/_community-header-theming.scss

```diff
@@ -0,0 +1,135 @@
+@import '../wds-functions/index';
+
+@mixin community-header-theming(
+	$header-background-color,
+	$color-text,
+	$color-links,
+	$color-page,
+	$color-page-border
+) {
+
+	// the same logic as $is-dark-wiki in oasis
+	$is-dark-theme: lightness($color-page) < 50;
+
+	$text-color: $wds-color-white;
+	@if luminance($header-background-color) > 0.3 {
```

Comment:
according to the guidelines we should not use any leading zeros https://github.com/Wikia/guidelines/blob/master/SCSS/CodingConventions.md#leadingzero